resources/views/brands/show.blade.php

```diff
@@ -0,0 +1,25 @@
+@extends('rapidez::layouts.app')
+
+@section('title', $meta_title->value() ?: $title)
+@section('description', $meta_description)
```

Comment:
Are those fields included in the brand blueprint?

Reply:
Now they are.